src/CMakeLists.txt

```diff
@@ -106,3 +111,35 @@ set_target_properties(box2d PROPERTIES
 
 source_group(TREE "${CMAKE_CURRENT_SOURCE_DIR}" PREFIX "src" FILES ${BOX2D_SOURCE_FILES})
 source_group(TREE "${CMAKE_CURRENT_SOURCE_DIR}/../include" PREFIX "include" FILES ${BOX2D_HEADER_FILES})
+
+install(
+  TARGETS box2d
+  EXPORT box2dTargets
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
```

Comment:
Do you really need those 3 lines? Aren't those the default values anyway?

Reply:
It's the default starting at CMake 3.14, Box2D defines the minimum version to 3.8. So it's necessary.

Reply:
Right.